goal_src/jak1/pc/features/speedruns.gc

```diff
@@ -133,6 +133,18 @@
   (and (-> *pc-settings* speedrunner-mode?)
        (name= category (enum->string speedrun-category (-> *speedrun-info* category)))))
 
+(defun enforce-speedrun-pc-settings ()
+  ;; Disable any active cheats
+  (set! (-> *pc-settings* cheats) (the-as pc-cheats #x0))
+  ;; ensure PS2 actor vis is enabled
+  (set! (-> *pc-settings* ps2-actor-vis?) #t)
+  ;; ensure FPS set to `60`
+  (when (!= (-> *pc-settings* target-fps) 60)
+    (set-frame-rate! *pc-settings* 60 #t)
+    )
```

Comment:
I got a bunch of console spam about "Tried to set unsupported video-mode" if I actually called this every frame, since `set-game-setting` resets `current` and assigns the specified value to `default`, but `(get-video-mode)` reads from `current`:
```lisp
(defmethod set-game-setting! ((obj pc-settings-jak1) (setting symbol) (value symbol))
  (case setting
    (('video-mode)
     (set! (-> *setting-control* current video-mode) #f)
     (set! (-> *setting-control* default video-mode) value))
    (('aspect-ratio) (set! (-> *setting-control* default aspect-ratio) value))
    (else (format #t "unknown setting  ~A (~A) to set-game-setting!" setting value))))